src/python/espressomd/interactions.pxd

```diff
@@ -298,45 +298,97 @@ IF SOFT_SPHERE:
                                         double a, double n,
                                         double cut, double offset)
 
+# not sure if this works when TABULATED is turned off ...
 IF TABULATED:
     cdef extern from "nonbonded_interactions/nonbonded_tab.hpp":
         int tabulated_set_params(int part_type_a, int part_type_b,
                                  double min, double max,
                                  vector[double] energy,
                                  vector[double] force)
 
-cdef extern from "bonded_interactions/bonded_interaction_data.hpp":
-    # Parameters for n-body tabulated potential (n=2,3,4).
-    cdef struct Tabulated_bond_parameters:
-        int type
-        TabulatedPotential * pot
+    cdef extern from "bonded_interactions/bonded_interaction_data.hpp":
+        # Parameters for n-body tabulated potential (n=2,3,4).
+        cdef cppclass Tabulated_bond_parameters:
+            Tabulated_bond_parameters()
+            Tabulated_bond_parameters(double min, double max,
+                                      const vector[double] & energy,
+                                      const vector[double] & force) except +
+            int type
+            TabulatedPotential * pot
```

Comment:
Should this raw pointer become a `shared_ptr[TabulatedPotential] pot` with the appropriate `from libcpp.memory cimport shared_ptr` at the top of the Cython file?